src/core/transaction.js

```diff
@@ -435,6 +435,15 @@ Transaction.prototype.signingHash = function(testnet) {
   return this.hash(testnet ? 'HASH_TX_SIGN_TESTNET' : 'HASH_TX_SIGN');
 };
 
+Transaction.prototype.signingData = function() {
+  return this._signingData('HASH_TX_SIGN').buffer;
+};
+
+Transaction.prototype._signingData = function(prefix) {
```

Comment:
Kinda pointless taking the prefix, given test net functionality is removed from rippled ...